src/main/java/hudson/plugins/ec2/EC2AbstractSlave.java

```diff
@@ -560,6 +564,11 @@ public String getPrivateDNS() {
         return privateDNS;
     }
 
+    public String getInstanceType() {
+        fetchLiveInstanceData(false);
+        return instanceType;
+    }
```

Comment:
I do not think this is needed if the goal is for getting it from Groovy scripts.

Reply:
I think it's better to have it - mainly for the consistency and to make it exposable for other plugins.